packages/conf-bap-llvm/conf-bap-llvm.1.2/files/configure

```diff
@@ -9,7 +9,10 @@
 # Note - suffix `-mp-` stands for macports, and will work on macs
 # if used with macports.
 
-config=${LLVM_CONFIG}
+config=`opam config var llvm-config 2>/dev/null` || true
```

Comment:
are you sure that interpolation doesn't work in this context? If it doesn't then it would be better to specify this file in the subst list, instead of  calling to opam everywhere and doing all these dev/null redirections.